packages/terra-modal/src/Modal.scss

```diff
@@ -0,0 +1,34 @@
+@import '~terra-mixins';
+
+.terra-Modal {
+  background-color: $terra-white;
```

Comment:
Use rem values for these pixel values. 
With the 14px root font, 10px = 0.714rem. 5px = 0.357rem.